### ISSUE TITLE
To embed this frame in your document, the response needs to enable the cross-origin embedder policy by specifying the following response header: Cross-Origin-Embedder-Policy: require-corp

### ISSUE THREAD

**Issue by linuxd3v** [QwikDev/partytown#255]
**Describe the bug**
To use atomics - I  add these headers to the html responses:
```
Cross-Origin-Embedder-Policy: credentialless   
Cross-Origin-Opener-Policy: same-origin
```
However - even without enabling partytown - this already causes some errors with some third party scripts.  I'm seeing these errors in the console:

> To embed this frame in your document, the response needs to enable the cross-origin embedder policy by specifying the following response header: Cross-Origin-Embedder-Policy: require-corp

These errors happen seemingly when third party scripts try to embed some html docs. 
Is this a known issue?

**Expected behavior**
no errors in the console

**Partytown version**
0.7.0

**Screenshots**
![image](https://user-images.githubusercontent.com/3410509/190665506-3e5f4840-d76e-4f42-bad5-fd8520a214df.png)

**Comment by Paul-Hebert** [comment]
I'm running into a similar issue with embedding iframes (e.g. code sandbox or codepen.)

My impression is that for third party iframes/scripts to work properly, those third parties would need to set CORS headers on their documents. Is that accurate?

I'm searching for solutions but may have to disable these headers for now to fix these embedded iframes.

(By the way, Partytown is awesome! Thanks for the great tool!)

**Comment by maplessmann** [comment]
We're also having the same issue

**Comment by taraskhvyl** [comment]
is there any solution here?

**Comment by mhevery** [comment]
Yes, atomics are very restrictive, to the point of being almost unusable. I don't think there is a good solution here...

**Comment by mrxbox98** [comment]
> Yes, atomics are very restrictive, to the point of being almost unusable. I don't think there is a good solution here...

Yeah. I switched over to atomics and my Stripe integration for payments stopped working. Maybe there is something else that I needed to enable, but it seems that currently service workers are the best option.

**Comment by gioboa** [comment]
That error is the Atomics mode requirement, not a bug: it needs Cross-Origin-Embedder-Policy / Cross-Origin-Opener-Policy headers on your origin.
See https://partytown.qwik.dev/atomics closing as expected behavior.
